BUILD.md

```diff
@@ -24,3 +24,17 @@ tito tag
 ```
 tito build --rpm
 ```
+
+## Build openshift-ansible container image
```

Comment:
This file starts with

```markdown
# openshift-ansible RPM Build instructions
 
We use tito to make building and tracking revisions easy.
```

Thus without reading through all of the file, I'd think of it as containing instructions for building an RPM.

Maybe we can update the introduction above, or simply keep all the information about the image in a single file?